lib/mock-component/test-components/custom-form-control.component.ts

```diff
@@ -0,0 +1,92 @@
+// sample control value accessor copied from
+// https://alligator.io/angular/custom-form-control/
```

Comment:
I'd prefer the test component not being from an outside project.  It'd be preferable that the example component just implements what you are trying to test.  No need for styles, stars, opacity, comments, or any additional logic.  Do you mind stripping all of that out?

Reply:
Sure, happy to strip it out. The reason I went that direction was to try to highlight a real world use case, for custom components with control value accessor, and this is the classic example.